src/styles/components/_forms.scss

```diff
@@ -81,6 +89,44 @@ select.form-control[multiple] {
 }
 
 // Form validation
+@mixin form-validation-state($state, $data) {
```

Comment:
Should we add mixin with in the forms, since we don't do it with variables? 

I think that we should do the same solution for both mixin and variables,  e.g add only global variables in variable.scss and have variables only related with the element inside the element.scss